pandas/core/generic.py

```diff
@@ -1153,34 +1155,24 @@ def rename_axis(
     ) -> Self | None:
         ...
 
-    def rename_axis(
-        self,
-        mapper: IndexLabel | lib.NoDefault = lib.no_default,
-        *,
-        index=lib.no_default,
-        columns=lib.no_default,
-        axis: Axis = 0,
-        copy: bool_t | None = None,
-        inplace: bool_t = False,
-    ) -> Self | None:
-        """
-        Set the name of the axis for the index or columns.
-
-        Parameters
-        ----------
+    @doc(
```

Comment:
While you're modifying this docstring could you just give both `rename_axis` separate docstrings? i.e. remove `@doc` from both?